katas/003_observability_in_chorus.md

```diff
@@ -3,7 +3,13 @@
 This Kata is a more technical one, where we learn how to set up some basic tooling to enable _Observability_ for Chorus.
 What is Observability you ask?  Checkout this blog post by New Relic as an intro: https://blog.newrelic.com/engineering/what-is-observability/
 
-We're assuming you've run the `quickstart.sh` script, or gone through the steps in [Kata 000 Setting up Chorus](katas/000_setting_up_chorus.md) already.
+We're assuming you've run the `quickstart.sh` script. If not, there are some commands to you up & running:
```

Comment:
I am not sure on this one... I worry about duplicating commands in multiple Kata's, and there is a sense that later kata's depend on earlier ones, you can't just skip around.   Could we BOTH reference the `000_setting_up_chorus.md` AND have the details here?  That way, if we end up adding more stuff to the setting up chorus, it's okay?

Reply:
Maybe, it makes sense to put the commands to set up the observability part in Kata 0. That way, you either run the `quickstart.sh` script (which has the observability part integrated) or you go through Kata 0 and set it up manually yourself.
Currently, (at least to my knowledge) the observability part is only in the `quickstart.sh` script. So if you set up Chorus yourself in Kata 0, you are unable to follow Kata 3 (this one) because you are missing some bits and pieces.
What do you think about that?

Reply:
Yeah, I should have those basic setup steps in Kata 0.   Can you make that change and then we are in good shape!

Reply:
Sounds good to me. You can find that in the latest commit https://github.com/querqy/chorus/pull/89/commits/94ecf8237a8c9b6b5f3a1fc36501fd4077427d25

Let me know if you find anything else we need to change.